test/BUILD.gn

```diff
@@ -74,6 +74,15 @@ group("other_unit_tests") {
   }
 }
 
+if (!is_android) {
+group("other_browser_tests") {
```

Comment:
`other_browser_tests` is not needed, you can add `"//brave/browser/ui/tabs/test:browser_tests"` directly as a dep of "browser_tests"

Reply:
Fixed.